models/runner/functions.py

```diff
@@ -180,14 +194,16 @@ def infer_to_csv(args):
     df_preds = parse_path(df_preds) # extract station, check, cam where possible from path
     df_preds = parse_exif(df_preds) # extract datetime, gps_long and gps_lat from exif if images
     df_preds = add_station_coords(df_preds, args.grid_file)
+
+    df_preds = add_output_date(df_preds)
     df_preds = add_output_coords(df_preds)
 
-    result = order_df(df_preds, ["path", "station", "check", "camera",\
-                                 "exif_datetime", "coordinates_long", "coordinates_lat",\
+    result = order_df(df_preds, ["path", "station", "check", "camera",
+                                 "date", "coordinates_long", "coordinates_lat",
```

Comment:
I expect the script to output all three columns for date (but they should be in the same, unified format), so that our Explore page can use `date`, but the user has access to all values and can manually correct them if needed. The same for GPS coordinates, we should write all 3 columns to the CSV even though the Explore page uses just one of them.

Reply:
Actually, all the columns were already being output. Still, I went ahead and unified their format, removed `--keep_scores` option (we always passed it anyway) and cleaned up the neighboring code.

Reply:
Great!